SchemaValidator/deployLambda.sh

```diff
@@ -0,0 +1,21 @@
+rm function.zip
+
+# Install package(s)
+source /home/bmatern/schemavalidationenv/bin/activate
```

Comment:
Although this is annoying, but path should be more generic. Maybe declare a variable for the path prefix and define it in the beginning? Then others may run this script with only adjusting a single line...

Reply:
Ok, done.